pkgs/tools/compression/bzip2/default.nix

```diff
@@ -48,7 +49,10 @@ in {
   outputs = [ "bin" "dev" "out" "man" ];
 
   configureFlags =
-    lib.optionals linkStatic [ "--enable-static" "--disable-shared" ];
+    lib.optional enableStatic "--enable-static" ++
+    lib.optional disableShared "--disable-shared";
+
+  dontDisableStatic = if enableStatic then true else null; # null to not cause rebuild vs historic versions that didn't have the attribute at all
```

Comment:
Is the purpose of this *only* to avoid a rebuild? If so, isn't it more important to keep the code simple?

Reply:
> Is the purpose of this _only_ to avoid a rebuild?

Yes, the usage of `else null` is.

With a rebuild, the entirety of nixpkgs would be rebuilt, so ofborg could not build it, this commit could not be cherry-picked easily, and it would have to target `staging`.

I am thinking that a followup PR to `staging` that removes `else null` could do the code simplification. But happy to hear others' opinions!

Reply:
we have no urgency here so please target staging straight away and avoid the rebuild reducing tricks